posts/2020-10-23-beta-20.0.0.12.adoc

```diff
@@ -0,0 +1,452 @@
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in Jakarta EE is the names of packages changing to accommodate the new `jakarta.*` namespace. In this Open Liberty beta, we have a number of new API Release Candidates to join the expanding library of supported Jakarta packages.
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* Jakarta WebSocket 2.0 (`websocket-2.0`; now with full CDI integration)
+* Enterprise Beans Remote Client 2.0 (`ejbRemoteClient-2.0`)
+* RESTful Web Services 3.0 (`restfulWS-3.0` and `restfulWSClient-3.0`)
+* Jakarta Server Faces 3.0 (`faces-3.0`)
+* Jakarta Connectors 2.0 (`connectors-2.0`)
+
+These join the Jakarta EE 9 features in link:https://openliberty.io/blog/?search=beta&key=tag[previous Open Liberty betas]:
+
+* Jakarta Enterprise Beans Remote 4.0 (`ejbRemote-4.0`)
+* Jakarta EE Application Client 9.0 (`javaeeClient-9.0`)
+* Jakarta Authentication 2.0 (`jaspic-2.0`)
+* Jakarta Authorization 2.0 (`jacc-2.0`)
+* Jakarta Persistence 3.0 (includes Eclipselink 3.0-RC1.) (`jpa-3.0`)
+* Jakarta XML Binding 3.0 (`jaxb-3.0`)
+* Jakarta Managed Beans 2.0 (`managedBeans-2.0`)
+* Jakarta Concurrency 2.0 (`concurrent-2.0`)
+* Jakarta Enterprise Beans Home 4.0 (`ejbHome-4.0`)
+* Jakarta Enterprise Beans Lite 4.0 (`ejbLite-4.0`)
+* Jakarta Bean Validation 3.0 (`beanValidation-3.0`)
+* Jakarta Contexts and Dependency Injection 3.0 (`cdi-3.0`)
+* JDBC 4.2 & 4.3 (`jdbc-4.2` & `jdbc-4.3`)
+* Jakarta Transactions 2.0 (`transaction-2.0`)
+* Jakarta JSON Binding 2.0 (`jsonb-2.0`)
+* Jakarta JSON Processing 2.0 (`jsonp-2.0`)
+* Jakarta Servlet 5.0 (`servlet-5.0`)
+* Jakarta Server Pages 3.0 (`jsp-3.0`)
+* Jakarta Expression Language 4.0 (`el-4.0`)
+
+Enable the Jakarta EE 9 beta features in your app's `server.xml`. You can enable the individual features you want or you can just add the Jakarta EE 9 convenience feature to enable all of the Jakarta EE 9 beta features at once:
+
+=== Try it now
+
+To try out these Jakarta EE 9 features on Open Liberty in a lightweight package, just update your build tools to pull the Open Liberty Jakarta EE 9 Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty.beta</groupId>
+    <artifactId>openliberty-jakartaee9</artifactId>
+    <version>20.0.0.12-beta</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-jakartaee9', version: '[20.0.0.12-beta,)'
+}
+----
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[source, xml]
```

Comment:
You've grabbed the dodgy version of Jakub's template (he's fixing it).
Move lines 412 (`[source,xml]`) to 426 (`----`) up to immediately before the `=== Try it now` heading (just after the paragraph that starts "Enable the Jakarta EE 9 features...".

This section is the server.xml entries that you use to enable the Jakarta EE 9 features. They shouldn't be down here after the Maven etc info.